src/core/interp.c

```diff
@@ -3759,6 +3759,10 @@ void MVM_interp_run(MVMThreadContext *tc, void (*initial_invoke)(MVMThreadContex
                 GET_REG(cur_op, 0).o = MVM_proc_clargs(tc);
                 cur_op += 2;
                 goto NEXT;
+            OP(getsignals):
+                GET_REG(cur_op, 0).o = MVM_io_get_signals(tc);
+                cur_op += 2;
+                goto NEXT;
```

Comment:
We try to keep the order of implementations in this file following the order of ops in `oplist`; please could you move it?

Reply:
Absolutely. I will move this when I revise later tonight.

Reply:
Moved this to after "getppid" op in the last commit. Not sure why GH doesn't indicate that this has changed like the other items.